hv.log.pd

```diff
@@ -0,0 +1,30 @@
+#N canvas 980 405 300 239 10;
+#X obj 14 22 inlet~;
+#X obj 14 212 outlet~;
+#X obj 14 189 *~ 1;
+#N canvas 620 157 160 113 @hv_obj 0;
+#X obj 16 9 inlet~;
+#X obj 16 76 outlet~;
+#X obj 16 54 log~;
+#X obj 37 31 sig~ 2;
+#X connect 0 0 2 0;
+#X connect 2 0 1 0;
+#X connect 3 0 2 1;
+#X restore 14 70 pd @hv_obj __log2~f;
+#X obj 35 116 log;
+#X obj 35 163 /;
+#X msg 35 140 0.693147 \$1;
+#X obj 14 46 max~ 0;
+#X text 61 46 [0 \, inf];
+#X obj 151 22 inlet;
+#X obj 151 44 hv.dispatch \$0 base 2;
```

Comment:
Not sure about appending `base` to the message. I know it's explicit but there's really only one argument it supports and it can be cumbersome to do this frequently.